.github/workflows/nightly.yml

```
@@ -55,14 +55,14 @@ jobs:
         pip install .[test]
         pip install --upgrade setuptools wheel
```

```suggestion
        pip install --upgrade setuptools wheel setuptools_scm
```
`no_local_version=$(python -m setuptools_scm | cut -d "+" -f 1)` will not work without it

---

good catch!